pkgs/applications/misc/cotp/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  AppKit,
+  libxcb,
+}:
+rustPlatform.buildRustPackage rec {
+  name = "cotp";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "replydev";
+    repo = "cotp";
+    rev = "v${version}";
+    sha256 = "sha256-SbS+ch7/45kZb49jW2mnRWQruLrfrNfeZFqEPHQKGUg=";
+  };
+
+  cargoSha256 = "sha256-2lIR3K4/hr4XSmNGFd/dhwoFOtEB9KSnUrZkcaCyc9k=";
+
+  buildInputs =
+    lib.optionals stdenv.isLinux [libxcb]
+    ++ lib.optionals stdenv.isDarwin [AppKit];
+
+  meta = with lib; {
+    homepage = "https://github.com/replydev/cotp";
+    description = "Trustworthy, encrypted, command-line TOTP/HOTP authenticator app with import functionality";
+    license = licenses.gpl3;
```

Comment:
Please change according to [this](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206).